tests/view_tests/tests/test_default_content_type.py

```diff
@@ -0,0 +1,44 @@
+import sys
```

Comment:
In retrospect, choose a naming that describes the deprecated functionality could be more useful.